README.md

```diff
@@ -272,6 +272,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 ## ORM/ODM Extensions
  * [active_record.cr](https://github.com/waterlink/active_record.cr) - Active Record pattern implementation
  * [amethyst-model](https://github.com/drujensen/amethyst-model) - ORM Model for the Amethyst Framework
+ * [crecto](https://github.com/fridgerator/crecto) - Database wrapper for Crystal, based on Ecto
```

Comment:
Please omit `for Crystal` in description. Everyone knows that this library is for Crystal if it is a part of this list.